### ISSUE TITLE
updateuid target does not exist in multitarget project

### ISSUE THREAD

**Issue by abbweiman** [dotnet/wpf#9991]
### Description

The `updateuid` target is no longer found in a SDK-style WPF project with multi targeting enabled.

### Reproduction Steps

1. Create a WPF project. Change `TargetFramework` to `TargetFrameworks`:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFrameworks>net6.0-windows</TargetFrameworks>
    <UseWPF>true</UseWPF>
  </PropertyGroup>
</Project>
```

**Note**: It doesn't matter if it's actual multi targeting or what's inside the TargetFrameworks. It's enough to add the 's'!

2. `msbuild /t:updateuid MyWpfProject.csproj`

### Expected behavior

No error and uids are updated in xaml files.

### Actual behavior

`error MSB4057: The target "updateuid" does not exist in the project.`

### Regression?

Yes! This has worked for years until recently.

### Known Workarounds

None.

### Impact

This is stopping us from updating translations in a big project.

### Configuration

Visual Studio 17.11.5
MSBuild 17.11.9

**Comment by lindexi** [comment]
The `UpdateUid` still here...

https://github.com/dotnet/wpf/blob/cbbd92cde6a3c1103ab71fe5c453ec8e017bfb7d/src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft.WinFX.targets#L882

My Visual Studio version is not 17.11.5

**Comment by dipeshmsft** [comment]
So @lindexi, from your comment it seems like this never worked with `TargetFrameworks` property set, am I right in my understanding ?
If this is the case then this issue should not be considered a regression. @abbweiman, your thoughts on this ?

**Comment by lindexi** [comment]
@dipeshmsft Did we ever change the structure of `Microsoft.WinFX.targets`, such as it being applicable to TargetFrameworks in the past but not anymore after a certain update? I tried to review the code change history, but I didn't find anything useful.

**Comment by dipeshmsft** [comment]
@lindexi, I don't think so, there have been few changes, but they mostly concerned with source generators.

**Comment by lindexi** [comment]
@dipeshmsft  Thank you for reminding me. That could be the reason.
